cts/cts-cli.in

```diff
@@ -1159,6 +1159,18 @@ class DatesRegressionTest(RegressionTest):
                  "iso8601 -d '2009-01-31 00:00:00Z' -D P2M -E '2009-03-31 00:00:00Z'"),
             Test("2009-01-31 + 3 Months",
                  "iso8601 -d '2009-01-31 00:00:00Z' -D P3M -E '2009-04-30 00:00:00Z'"),
+            Test("Test correct immediate plus-sign at timezone parsing",
+                 "iso8601 -d 2019-04-25T21:42:54+0200 -S"),
+            Test("Like the previous, with exotic timezone to avoid false negative",
+                 "iso8601 -d 2019-04-25T21:42:54+0420 -S"),
+            Test("Whitespace 1",
+                 "iso8601 -d 2019-04-25 T21:42:54+0420 -S"),
+            Test("Whitespace 2",
+                 "iso8601 -d 2019-04-25 T 21:42:54+0420 -S"),
+            Test("Whitespace 3",
+                 "iso8601 -d 2019-04-25 T 21:42:54 +0420 -S"),
+            Test("Whitespace 4",
+                 "iso8601 -d 2019-04-25 T 21:42:54 + 0420 -S"),
```

Comment:
Put single quotes around all the time strings in Whitespace 1 through Whitespace 4.  Otherwise, those will be seen as all separate arguments by iso8601.